samples/sampler/.qa-storybook/main.js

```diff
@@ -1,23 +1,42 @@
 /* global __dirname */
 
 const webpack = require('@enact/storybook-utils/configs/webpack');
+const {loadCsf} = require('@storybook/csf-tools');
+const {readFileSync}  = require('fs');
 
 module.exports = {
 	core: {
-		builder: 'webpack5',
 		disableTelemetry: true
 	},
 	features: {
 		postcss: false,
 		warnOnLegacyHierarchySeparator: false
 	},
-	framework: '@storybook/react',
+	framework: {
+		name: '@storybook/react-webpack5',
+		options: {}
```

Comment:
Could you remove this empty options? I thought it was removed.

Reply:
ok